libvmi/os/windows/core.c

```diff
@@ -610,6 +610,11 @@ void windows_read_config_ghashtable_entries(char* key, gpointer value,
         goto _done;
     }
 
+    if (strncmp(key, "win_init_vcpu", CONFIG_STR_LENGTH) == 0) {
```

Comment:
The init vCPU should be passed in as `VMI_INIT_DATA_INIT_VCPU` instead of via the ghashtable since that only works with `VMI_CONFIG_GHASHTABLE`

Reply:
I think if the appropriate key would be added to ` libvmi/config/grammar.y` then it will also work with `VMI_CONFIG_GLOBAL_FILE_ENTRY` and `VMI_CONFIG_STRING`.

Also `vmi_init_os` doesn't accept `vmi_init_data_t *init_data` as an argument. It is only accepted by `vmi_init` or `vmi_init_complete`. In case of `vmi-win-offsets` tool, we don't yet know KPGD value at the moment when we call `vmi_init()`.

Reply:
I've added an entry to `grammar.y`. I think it would be much easier this way, as `vmi_init_data_t` can not be easily used here, for reasons I've outlined above.

Reply:
True.. but then the problem is this can't be used with VMI_CONFIG_JSON_PATH.. I think it would make more sense to just update the OS init code in LibVMI to try all vCPUs instead of just vCPU 0.